docs/responses/special_rules.rst

```diff
@@ -0,0 +1,91 @@
+SPECIAL
+^^^^^^^
+
+As PyFunceble grows, I thought that a bit of filtering for special
+cases would be great.
+
+So I introduced the SPECIAL source.
+
+For any new suggestion of domains where a special rule can enhance PyFunceble
+please either open an new `issue`_ or make you
+`contribution <../contributing/index.html#contribute>`_ directly into the git.
+
+.. note::
+    Please consider all 3 digits number that are listed in this section as
+    the HTTP status code catches by
+    :func:`PyFunceble.lookup.http_code.HTTPCode.get`.
+
+.. note::
+    If you do not want to use these rules? You can disable these flags to
+    disable the feature.
+
+    * `--special-lookup <../usage/index.html#special-lookup>`_ arguments from the
+      CLI.
+    * :code:`special: True` into your local configuration file.
```

Comment:
Is `lookup.special: True` not better ? 🤔

Reply:
IF, then follow the flow/order `special.lookup`

Reply:
what I meant, when we flatten:

```yaml
lookup:
    special: False
```

it becomes `lookup.special`.

Reply:
it true... I can work with that logic :smile: and it would make more sense

but the entire section needs a second hand, so I'm open for changes, before I rewrite....

```rst
.. note::
    If you do not want to use these rules? You can disable these flags to
    disable the feature.

    * `--special-lookup <../usage/index.html#special-lookup>`_ arguments from the
      CLI.
    * :code:`special: True` into your local configuration file.
```